src/prefixes.json

```diff
@@ -1,36 +1,37 @@
 {
-	"@context": {
-		"BFO": "http://purl.obolibrary.org/obo/BFO_",
-		"CHEBI": "http://purl.obolibrary.org/obo/CHEBI_",
-		"CL": "http://purl.obolibrary.org/obo/CL_",
-		"CMO": "http://purl.obolibrary.org/obo/CMO_",
-		"COB": "http://purl.obolibrary.org/obo/COB_",
-		"EFO": "http://purl.obolibrary.org/obo/EFO_",
-		"GCS": "https://purl.ihccglobal.org/GCS_",
-		"GE": "https://purl.ihccglobal.org/GE_",
-		"GECKO": "http://purl.obolibrary.org/obo/GECKO_",
-		"GO": "http://purl.obolibrary.org/obo/GO_",
-		"HP": "http://purl.obolibrary.org/obo/HP_",
-		"IAO": "http://purl.obolibrary.org/obo/IAO_",
-		"IHCC": "https://purl.ihccglobal.org/",
-		"KOGES": "https://purl.ihccglobal.org/KoGES_",
-		"OGMS": "http://purl.obolibrary.org/obo/OGMS_",
-		"MAELSTROM": "https://purl.ihccglobal.org/MAELSTROM_",
-		"MMO": "http://purl.obolibrary.org/obo/MMO_",
-		"MONDO": "http://purl.obolibrary.org/obo/MONDO_",
-		"NCBITaxon": "http://purl.obolibrary.org/obo/NCBITaxon_",
-		"NCIT": "http://purl.obolibrary.org/obo/NCIT_",
-		"OBI": "http://purl.obolibrary.org/obo/OBI_",
-		"obo": "http://purl.obolibrary.org/obo/",
-		"oboInOwl": "http://www.geneontology.org/formats/oboInOwl#",
-		"owl": "http://www.w3.org/2002/07/owl#",
-		"PATO": "http://purl.obolibrary.org/obo/PATO_",
-		"PDRO": "http://purl.obolibrary.org/obo/PDRO_",
-		"PR": "http://purl.obolibrary.org/obo/PR_",
-		"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
-		"SAPRIN": "https://purl.ihccglobal.org/SAPRIN_",
-		"STATO": "http://purl.obolibrary.org/obo/STATO_",
-		"UBERON": "http://purl.obolibrary.org/obo/UBERON_",
-		"VZ": "https://purl.ihccglobal.org/VZ_"
-	}
-}
+    "@context": {
```

Comment:
Why is the DIFF so large? JSON indentation? @beckyjackson

Reply:
Same as above. I think after the first merge, the diff will be nicer. This was all hand-edited before.

Reply:
perfect thx